packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -314,6 +314,8 @@ export default ({
       order.extraAttributes.authorisationUrl
     ) {
       return order
+    } else if (order.state === 'FAILED') {
+      return order
```

Comment:
Since this is just doing `return order` just like line 316 you can just add `|| order.state === 'FAILED'` to the previous if statement.